EIPS/eip-6049.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: 6049
+title: Deprecate SELFDESTRUCT
+description: Deprecate SELFDESTRUCT by discouraging its use and warning about a potential future behavior change.
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/deprecate-selfdestruct/11907
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-11-27
+---
+
+## Abstract
+
+This EIP deprecates the `SELFDESTRUCT` opcode and warns against its use. A breaking change to this functionality is likely to come in the future.
+
+## Motivation
+
+Discussions about how to change `SELFDESTRUCT` are ongoing. But there is a strong desire to do *something*.
```

Comment:
```suggestion
Discussions about how to change `SELFDESTRUCT` are ongoing. But there is a strong consensus that *something* will change.
```
This language should be stronger.

Reply:
I agree. But if anyone asks me to prove there is consensus then I need to cite a lot to defend that.